### PR TITLE
Planning and migration strategy

### PR DESCRIPTION
**Pull request by darienmt**

### CODE REVIEW

File: content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Planning and executing a migration strategy
+
+## Key points
+
+* It is a journey: do not jump into the water to try how deep it is [What Is Your Cloud Migration Strategy?](https://www.cloudstrategymag.com/articles/86667-what-is-your-cloud-migration-strategy)
```
Comment (vkhazin) [inline content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md:5]: It is a journey: do not jump head first into the water to test how deep it is```<a href="https://www.cloudstrategymag.com/articles/86667-what-is-your-cloud-migration-strategy" target="_blank">What Is Your Cloud Migration Strategy?</a>```

File: content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+* Done do it all by yourself: Use open source tools (Ex. [Terraform](https://www.terraform.io/)) or vendor specific tools for migration and orchestration (Ex. [Which Microsoft tools can help me migrate to Azure?](http://searchcloudcomputing.techtarget.com/answer/Which-Microsoft-tools-can-help-me-migrate-to-Azure))
```
Comment (vkhazin) [inline content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md]: Links in a new window!

File: content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+* Your infrastructure can be manipulated by code using API [Cloud Migration Strategies for Enterprises](https://dzone.com/articles/cloud-migration-strategies-for-enterprises)
```
Comment (vkhazin) [inline content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md]: Links in a new window!

File: content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+* **People and culture** are always the most significant barrier
```
Comment (vkhazin) [inline content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md]: **People and culture** are always the most likely obstacles
> A reference would be nice here

File: content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+* Plan, but always expect the unexpected [How to Survive 4 Cloud Horror Story Scenarios](https://www.cio.com/article/2460967/cloud-computing/how-to-survive-4-cloud-horror-stories.html)
```
Comment (vkhazin) [inline content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md]: links in a new window

**Review by vkhazin — CHANGES_REQUESTED** [review]
Please review my comments, welcome to disagree

File: content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md (hunk 1 above)
Comment (darienmt) [reply content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md:5]: Links are opening on a new windows.

File: content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md (hunk 2 above)
Comment (darienmt) [reply content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md]: Links are opening on a new windows.

File: content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md (hunk 3 above)
Comment (darienmt) [reply content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md]: Links are opening on a new window.

File: content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md (hunk 5 above)
Comment (darienmt) [reply content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md]: Links are opening on a new window.

File: content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md (hunk 4 above)
Comment (darienmt) [reply content/chapter-01/PlanningAndExecutingMigrationKeyPoints.md]: References added.

**Pull request merged**